test/runtests.jl

```diff
@@ -330,6 +330,17 @@ end
             @test f(0,1) ≈ f(0,Complex{Float32}(1))
         end
     end
+    @testset "besselh order range" begin
```

Comment:
Could you add a test here for the `ArgumentError` case noted above?

Reply:
Done.